pkg/codegen/hcl2/binder_schema.go

```diff
@@ -171,6 +171,9 @@ func (b *binder) schemaTypeToTypeImpl(src schema.Type, seen map[schema.Type]mode
 			if !prop.IsRequired {
 				t = model.NewOptionalType(t)
 			}
+			if prop.ConstValue != nil {
+				t = model.NewConstType(t, prop.ConstValue)
+			}
```

Comment:
This seems reasonable to me, as it will improve our ability to type check PCL programs against a schema. that contains const-valued props.